spec/support/features/session_helper.rb

```diff
@@ -16,14 +16,14 @@ def expect_email_invalid(page)
       expect(page).to have_content t('sign_up.email.invalid_email_alert_inline')
     end
 
-    def choose_another_security_option(option)
+    def choose_another_security_option(option = nil)
       accept_rules_of_use_and_continue_if_displayed
 
       click_link t('two_factor_authentication.login_options_link_text')
 
       expect(current_path).to eq login_two_factor_options_path
 
-      select_2fa_option(option)
+      select_2fa_option(option) if option
```

Comment:
making `option` optional turns this helper method into just clicking "accept rules of use" so maybe can we just update the callsite that passes nil to just do `accept_rules_of_use_and_continue_if_displayed` and `click_link t('two_factor_authentication.login_options_link_text')`  instead?

Reply:
Sure, I can do that. The thinking was a parameter-less call to "choose another security option" could be interpreted as "bring me to the MFA selection screen", but in retrospect I think the difference in some calls _confirming a selection_ vs. others bringing to the prompt screen could be confusing.

Reply:
Updated in b61c5c871.